python/ray/__init__.py

```diff
@@ -24,6 +26,23 @@
     from ctypes import CDLL
     CDLL(so_path, ctypes.RTLD_GLOBAL)
 
+if (platform.system().lower() == "Linux".lower()
+    and "Microsoft".lower() in platform.release().lower()):
+    # WSL's /proc/meminfo has a bug that crashes Python; work around it
+    try:
+        import psutil._pslinux
+    except ImportError:
+        psutil = None
+    psutil_open_binary = None
+    try: psutil_open_binary = psutil and psutil._pslinux.open_binary
```

Comment:
what does this resolve to? I would expect a boolean but it's used as a function later? looks like too much python magic to me can you please try to simplify

Reply:
Ah, ok sure, but if I simplify this then I can't do the above. But it's supposed to resolve to either `open_binary` or `None`.